ironfish/src/rpc/routes/transactions/sendTransaction.ts

```diff
@@ -91,6 +92,24 @@ router.register<typeof SendTransactionRequestSchema, SendTransactionResponse>(
       )
     }
 
+    // Check whether amount and fee are valid or not
+    transaction.receives.map((receive) => {
```

Comment:
nit - we already iterate over `transaction.receives` twice in this function on lines 116 and 135. Maybe we should consolidate that logic to a single loop?